common/src/constants.rs

```diff
@@ -20,3 +20,8 @@ pub const REQUEST_TIMEOUT: Duration = Duration::from_secs(45);
 
 /// Alchemy free tier cups <https://github.com/alchemyplatform/alchemy-docs/blob/master/documentation/compute-units.md#rate-limits-cups>
 pub const ALCHEMY_FREE_TIER_CUPS: u64 = 330;
+
+/// Logged when an error is indicative that the user is trying to fork from a non-archive node.
+pub const NON_ARCHIVE_NODE_WARNING: &str = "\
```

Comment:
Actually we can move this to the anvil `config` file to colocate these things together instead of having this here.

Reply:
hmm can I import it to `forge` from there?

Reply:
The rationale behind moving it to where it's used is that we usually don't make errors constant, but rather just declare them as an inline string usually

Reply:
Hey @Evalir , thanks for the reply! Just to clarify: I should replace occurrences of `NON_ARCHIVE_NODE_WARNING` with the string literal? Note that `NON_ARCHIVE_NODE_WARNING` is used in 5 places in two different crates (`evm` and `anvil`)

Reply:
ahh, in this case then i think it's good to leave it as a constant.